docs/highlighters/regexp.md

```diff
@@ -12,11 +12,32 @@ To use this highlighter, associate regular expressions with styles in the
 
 ```zsh
 typeset -A ZSH_HIGHLIGHT_REGEXP
-ZSH_HIGHLIGHT_REGEXP+=('\bsudo\b' fg=123,bold)
+ZSH_HIGHLIGHT_REGEXP+=('^rm .*' fg="red",bold)
```

Comment:
Quotes around red aren't needed and are a little odd, but just a style nit.

Reply:
Well, that's because I also work with python. Thank you very much!